codex-ui/src/vue/composables/useTheme.ts

```diff
@@ -0,0 +1,143 @@
+import { useLocalStorage, createSharedComposable } from '@vueuse/core';
+import type { Ref } from 'vue';
+
+/**
+ * Available themes for the application.
+ */
+export enum Theme {
+  Classic = 'classic',
+  Crimson = 'crimson',
+  Grass = 'grass',
+  Red = 'red',
+  Violet = 'violet',
+}
+
+/**
+ * Available color mode values
+ */
+export enum ColorMode {
+  Light = 'light',
+  Dark = 'dark',
+}
+
+/**
+ * Local storage key for base theme value
+ */
+const LOCAL_STORAGE_KEY_THEME_BASE = 'theme_base';
+
+/**
+ * Local storage key for accent theme value
+ */
+const LOCAL_STORAGE_KEY_THEME_ACCENT = 'theme_accent';
+
+/**
+ * Local storage key for color mode value
+ */
+const LOCAL_STORAGE_KEY_COLOR_MODE = 'color_mode';
+
+/**
+ * Apply theme to the body element
+ *
+ * @param theme – theme to apply
+ * @param type – type of theme to apply
+ */
+function applyTheme(theme: Theme, type: 'base' | 'accent'): void {
```

Comment:
lets reorder params: type first and theme value next to it? It would look more straightforward